feat(docs):[TRI-1654] Add section in building block view for subproto… (#575)
catenax-ng/tx-item-relationship-service: docs/src/docs/arc42/building-block-view/level-2.adoc

|The DigitalTwinRegistryFacade calls the DigitalTwinRegistry to retrieve data form the AAS registry and transforms the response to internal data models.

|SubmodelFacade

> |The SubmodelFacade handles EDC contract negotiation, calls the EDC dataplane to retrieve data from the submodel server and transforms the response to internal data models.

Reviewer: ```suggestion
|The SubmodelFacade handles EDC contract negotiations and is responsible to request the EDC dataplane to retrieve data from the submodel server and transforms the response to the internal data models. <jobResponse>
```

> |The SubmodelFacade handles EDC contract negotiation, calls the EDC dataplane to retrieve data from the submodel server and transforms the response to internal data models.

Author: ```suggestion
|The SubmodelFacade handles EDC contract negotiations and is responsible for the EDC dataplane requests to retrieve data from the submodel servers.
```
